System-Automation-Scripts/automatedIMDB.py

```diff
@@ -0,0 +1,67 @@
+#IT IS A MINI-IMDB PROJECT THAT PROVIDES INFORMATION ABOUT ANY MOVIE OR TV SERIES WHEN SEARCHED.
```

Comment:
Nicely coded project. Please try and get the code formatting to PEP8 format for improved readability.